src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingTemplate.cs

```diff
@@ -36,6 +37,7 @@ internal BindingTemplate(string pattern, IReadOnlyList<BindingTemplateToken> tok
             _pattern = pattern;
             _tokens = tokens;
             _ignoreCase = ignoreCase;
+            _parameterResolver = parameterResolver ?? new DefaultParameterResolver();
```

Comment:
Eventually we'll want this DefaultParameterResolver to be added to the JobHost service collection so it can be replaced entirely. There are some hurdles to that currently.  First, there isn't any way to get to the services collection from this point in the code, and most existing callers won't be passing a resolver in. Currently its only the extensions build on the new extensibility model. So until we can put a consistent story in place across the board for this, the default resolver remains internal.